### PR TITLE
Update trove classifiers

### PR DESCRIPTION
**Pull request by ShreehariVaasishta**
This pull request refers to the issue https://github.com/djangopackages/djangopackages/issues/676.

### CODE REVIEW

File: package/utils.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from trove_classifiers import deprecated_classifiers
```
Comment (jefftriplett) [inline package/utils.py:1]: I think we want: 

```suggestion
from trove_classifiers import classifiers
```

File: package/utils.py [hunk 2]
```diff
@@ -66,7 +67,7 @@ def normalize_license(license):
     """
     if license is None:
         return "UNKNOWN"
-    if license.strip() in settings.LICENSES:
+    if license.strip() in deprecated_classifiers:
```
Comment (jefftriplett) [inline package/utils.py:70]: See above change too: 

```suggestion
    if license.strip() in classifiers:
```

**Review by jefftriplett — CHANGES_REQUESTED** [review]
This is great. I think we just need to apply the suggestions and see how tests do.

File: package/utils.py (hunk 1 above)
Comment (ShreehariVaasishta) [reply package/utils.py]: Got it, thanks for the help.

**Review by jefftriplett — APPROVED** [review]
🎉 great work. Thank you for your contribution!

**Pull request merged**